components/Theme.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+
+const Theme = (theme?: string) => {
+  const [isReady, setIsReady] = useState(false);
+  useEffect(() => {
+    if (!theme && window.matchMedia("(prefers-color-scheme: dark)").matches) {
+      document.documentElement.classList.add("dark");
+    } else {
+      document.documentElement.classList.add(theme);
+    }
+    setIsReady(true);
+  }, []);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'theme'. Either include it or remove the dependency array.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  }, [theme]);
```